extensions/websockets/client/runtime/src/main/java/io/quarkus/websockets/BearerTokenClientEndpointConfigurator.java

```diff
@@ -53,4 +54,8 @@ public List<Class<? extends Decoder>> getDecoders() {
     public Map<String, Object> getUserProperties() {
         return Collections.emptyMap();
     }
+
+    public SSLContext getSSLContext() {
+        return null;
+    }
```

Comment:
@sberyozkin this one is interesting. `ClientEndpointConfig` introduces a new `getSSLContext()` method that should return a non-null SSLContext if you want to usee wss.
That being said, I have no idea how you are supposed to use this one? Should people only use the configurator?